blog/12-12-2022-advance-hardhat-configration-on-celo-using-plugins/advance-hardhat-configration-on-celo-using-plugins.md

```diff
@@ -0,0 +1,751 @@
+---
```

Comment:
This file should not be here, There are two articles in one PR, which is not allowed , you have to create in different branches

Reply:
Hi, @ Ishanpathak1 You're reviewing the wrong PR.  Please take a look at the PR title and check the exact one with the same title.